.eslintrc.json

```diff
@@ -5,7 +5,8 @@
   "extends": [
     "@code-pushup/eslint-config/typescript",
     "@code-pushup/eslint-config/node",
-    "@code-pushup/eslint-config/jest"
+    "@code-pushup/eslint-config/jest",
+    "@code-pushup/eslint-config/vitest"
```

Comment:
When I open the project in my IDE, ESLint complains about the test filename.

The vitest/consistent-test-filename requires a pattern for all test files.
I see that there are still some leftover `spec` files. So for `.spec.ts` or `.test.ts`, the following setting will fix the ESLint complaints. Once the `spec` files are renamed, the pattern can be even more strict:

```
{
      "files": ["*.test.ts", "*.spec.ts"],
      "rules": {
        "vitest/consistent-test-filename": {
          "pattern": {
            "format": "regex",
            "default": ".*\\.(test|spec)\\.ts$"
          }
        }
      }
    }
```

Reply:
We have decided to use test.ts and start transitioning to unit, integration, and e2e. Lets just rename all spec to test. 

Also i think it currently is set to only find files that end with test.

Reply:
Some rules require an explicit setting for most projects. `vitest/consistent-test-filename` is one of them as illustrated in the `@code-pushup/eslint-config` documentation for [`vitest`](https://github.com/code-pushup/eslint-config/blob/main/docs/vitest.md#%EF%B8%8F-setup). Without this explicit configuration, all `.test.ts` files look like this in an IDE with linter turned on:
![image](https://github.com/push-based/user-flow/assets/6306671/16239e76-a41c-4064-8fc0-fdb12b1c86d2)

So the snippet in the message above temporarily allows both and once you only use one, you can narrow the rule down to the regexp used in the `eslint-config` example, aka `{ "pattern": ".*\\.(unit|integration|e2e)\\.test\\.ts$" }`